tests/perl/testfiles/.perlcriticrc

```diff
@@ -0,0 +1,8 @@
+severity  = 5
+# for signatures
+[-Subroutines::ProhibitSubroutinePrototypes]
+
+[-TestingAndDebugging::RequireUseStrict]
```

Comment:
@AbdealiJK so doing this (notice the `-` at the start of the line) removes the error shown (which is enabled by default). So this will pass the `bad_file` too, thereby verifying the config file usage.